test/generalizedgaussian.jl

```diff
@@ -0,0 +1,234 @@
+using Distributions
+using Test
+
+
+d = GeneralizedGaussian() # mean zero, scale √2, shape 2.
```

Comment:
to avoid putting everything in global scope, use a testset bloc:
```julia
@testset "generalized gaussian" begin
    d = GeneralizedGaussian() # mean zero, scale √2, shape 2.
    # other stuff
end
```